include/openthread/thread.h

```diff
@@ -182,6 +182,23 @@ typedef struct otMleCounters
     uint16_t mParentChanges;
 } otMleCounters;
 
+/**
+ * This structure represents the parent response stats data which a client can request on every successful parent
+ * response frame handling.
+ *
+ */
+typedef struct otThreadParentResponseStats
```

Comment:
Not really "stats", but the contents of the Parent Response. Maybe otThreadParentResponse{Info,Data}?

Reply:
updated